### ISSUE TITLE
Pagination not removed when using sortablegridfield

### ISSUE THREAD

**Issue by cam-findlay** [silverstripe/silverstripe-dms#16]
Due to an API change in the SortableGridField module the method setUsePagination no longer exists. This leads to the pagination not being disabled when we enable sorting in the CMS for documents attached to a page in DMS.

see DMSSiteTreeExtension.php line: 71

```php
        if(class_exists('GridFieldSortableRows')) {
            $sortableComponent = new GridFieldSortableRows('DocumentSort');
            //setUsePagenation method removed from newer version of SortableGridField.
            if(method_exists($sortableComponent,'setUsePagination')){
                $sortableComponent->setUsePagination(false)->setForceRedraw(true);
            }
            $gridFieldConfig->addComponent($sortableComponent);
        }
```

**Comment by robbieaverill** [comment]
@cam-findlay @HARVS1789UK is this still an issue in more recent versions of SS/DMS?

**Comment by robbieaverill** [comment]
I tested this yesterday - the GridFieldSortableRows lets you drag to the left and right of the page to move to the previous or next page. It seems to be a little bit buggy, works sometimes and not others, but from what I can see this is the replacement for the previous "setUsePagination" method etc. This functionality is enabled automatically when you've got the sortablegridfield module installed, so perhaps any issues about this need to be raised over there?

I tested against the 2.0 version of DMS, but imagine it's the same for 1.4
